src/include/cursor.h

```diff
@@ -211,33 +211,30 @@ struct __wt_cursor_btree {
 struct __wt_cursor_bulk {
 	WT_CURSOR_BTREE cbt;
 
-	WT_REF	*ref;			/* The leaf page */
-	WT_PAGE *leaf;
+	bool	 first_insert;		/* First insert, else check order  */
```

Comment:
@agorrod:

> Can you call these three out in a block saying that they are only used in the column store case?

The `first_insert` field is both row- and column-store; I'll move `first_insert` next to the `WT_ITEM last` field (which is also shared), and call the remaining two out in a block as only column-store support. I can move the reconciliation information to the end, too.